static/js/public/first-snap-flow.js

```diff
@@ -84,22 +84,61 @@ function getSnapCount(cb) {
   fetch("/snaps/api/snap-count")
     .then(r => r.json())
     .then(data => {
-      cb(data.count);
+      cb(data);
     });
 }
 
+function newArrayValue(arr1, arr2) {
```

Comment:
Not sure exactly if I read this function correctly, but could we name it more clearly to what it does? Something like `getArrayDiff` (if I get it right)?

Reply:
:+1: I had a complete mind blank when naming this function, thanks. Will change it